src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -7373,28 +7373,7 @@ static void Main(string[] args)
         }
     }
 }";
-            var expected = @"class Test
-{
-    void method()
-    {
-        static void Main(string[] args)
-        {
-            int v = 0;
-            for(int i=0 ; i<5; i++)
-            {
-                v = NewMethod(v, i);
```

Comment:
> No, there is proper brace matching (a closing brace) in this code. This is actual nesting.

I'm not comfortable making that case.  Take, for example:

```c#
x.
await T;
```

This is syntactically correct, but that doesn't mean the the syntax tree corresponds to what the user thinks is going on.  

Take the case where there is no trailing curly brace either.  Will that case work properly?  Or will your change make it so that code doens't work either?